### DISCUSSION TITLE
How a frame can resize to remaining height in a Body

### DISCUSSION THREAD

**Discussion by heejit** [cogentcore/core#1219] [General]
As suggested I am using frames to layout widgets.

I have created two frame

In 1st frame widget arrange horizontally.

2nd frame has TextEditor 

I want TextEditor to take remaining height and take full width.

**Comment by heejit** [comment]
It is working :
	
```
        frame2 := core.NewFrame(this.Body)
	frame2.Styler(func(s *styles.Style) {
		s.Grow.Set(1, 1)
	})
	this.ResultEditor = texteditor.NewEditor(frame2)
	this.ResultEditor.Styler(func(s *styles.Style) {
		s.SetReadOnly(true)
		s.Grow.Set(1, 1)
	})

```

**Reply by kkoreilly** [reply]
Yes, you need to set grow on any parent frame for it to work. You may also be interested in [splits](https://www.cogentcore.org/core/widgets/containers/splits), which make it easy to arrange elements in customizable ways.
